src/core/api/tcp_ext_api.cpp

```diff
@@ -79,4 +79,48 @@ otError otTcpCircularSendBufferDeinitialize(otTcpCircularSendBuffer *aSendBuffer
     return AsCoreType(aSendBuffer).Deinitialize();
 }
 
+int otTcpMbedTlsSslSendCallback(void *aCtx, const unsigned char *aBuf, size_t aLen)
+{
+    otTcpEndpointAndCircularSendBuffer *pair       = static_cast<otTcpEndpointAndCircularSendBuffer *>(aCtx);
+    otTcpEndpoint *                     endpoint   = pair->mEndpoint;
+    otTcpCircularSendBuffer *           sendBuffer = pair->mSendBuffer;
+    size_t                              bytes_written;
+    int                                 result;
+    otError                             error;
+
+    error = otTcpCircularSendBufferWrite(endpoint, sendBuffer, aBuf, aLen, &bytes_written, 0);
+    VerifyOrExit(error == OT_ERROR_NONE, result = MBEDTLS_ERR_SSL_INTERNAL_ERROR);
```

Comment:
Need this:

`#include <mbedtls/debug.h>
`

Reply:
Sorry, I'm not seeing why we need this `#include`?